android/README.md

```diff
@@ -0,0 +1,71 @@
+
+Android BluetoothLeGatt Sample
+===================================
+
+This sample demonstrates how to use the Bluetooth LE Generic Attribute Profile (GATT)
+to transmit arbitrary data between devices.
+
+Introduction
+------------
+
+This sample shows a list of available Bluetooth LE devices and provides
+an interface to connect, display data and display GATT services and
+characteristics supported by the devices.
+
+It creates a [Service][1] for managing connection and data communication with a GATT server
+hosted on a given Bluetooth LE device.
+
+The Activities communicate with the Service, which in turn interacts with the [Bluetooth LE API][2].
+
+[1]:http://developer.android.com/reference/android/app/Service.html
+[2]:https://developer.android.com/reference/android/bluetooth/BluetoothGatt.html
+
+Pre-requisites
+--------------
+
+- Android SDK 26
+- Android Build Tools v26.0.1
+- Android Support Repository
+
+Screenshots
+-------------
+
+<img src="screenshots/1-main.png" height="400" alt="Screenshot"/> <img src="screenshots/2-detail.png" height="400" alt="Screenshot"/> 
+
+Getting Started
+---------------
+
+This sample uses the Gradle build system. To build this project, use the
+"gradlew build" command or use "Import Project" in Android Studio.
+
+Support
```

Comment:
Update README to contain information relevant for the Able project.

Reply:
TODO